joint_names:
  - joint_a1
  - joint_a2
  - joint_a3
  - joint_a4
  - joint_a5
  - joint_a6
  - joint_a7

poses:

  pregrasp:
    positions:
      - -0.55
      - -0.10
      -  0.55
      -  1.15
      -  0.30
      - -0.20
      -  -0.30
    time_from_start: 3.0

  # grasp = pregrasp (logicamente diverso, cinematicamente uguale)
  grasp:
    positions:
      - -0.55
      - -0.10
      -  0.55
      -  1.15
      -  0.30
      - -0.20
      - -0.30
    time_from_start: 2.0
  
  lift:
    positions:
      - -1.80  
      -  0.30
      -  0.55
      -  1.15
      -  0.30
      - -0.20
      -  0.00
    time_from_start: 3.0
    
  pre_handover:
    positions:
      - -1.95
      - -0.70   
      -  0.30
      -  1.40
      -  0.00
      -  0.30
      - -0.30
    time_from_start: 2.5

  
  handover:
    positions:
      - -1.65
      - -1.50
      -  0.30
      -  1.20
      -  0.00
      -  1.20
      - -0.30
    time_from_start: 3.0

